client/src/ui/organisms/article-actions/index.scss

```diff
@@ -19,124 +19,6 @@
     }
   }
 
-  .article-actions-submenu {
```

Comment:
Nit: Hard to tell why this was moved, so this should have been a separate commit with an explanation.